tests/src/test_cases/widgets/test_msgbox.c

```diff
@@ -144,4 +144,38 @@ void test_msgbox_close_async_modal(void)
     TEST_ASSERT_NOT_NULL(msgbox);
 }
 
+void test_msgbox_content_auto_height(void)
+{
+    // If parent is NULL the message box will be modal
```

Comment:
We use `/**/` comments:
```suggestion
    /* If parent is NULL the message box will be modal */
```